rdrf/rdrf/settings.py

```diff
@@ -317,6 +319,8 @@
 CSRF_HEADER_NAME = env.get("csrf_header_name", 'HTTP_X_CSRFTOKEN')
 CSRF_TRUSTED_ORIGINS = env.getlist("csrf_trusted_origins", ['localhost'])
 
+# Content Security Policy
+CSP_DEFAULT_SRC = tuple(filter(None, ["'self'", "'unsafe-inline'", get_static_url_domain(env.get("STATIC_URL", ""))]))
```

Comment:
This can be a tuple OR a list. So maybe
``[x for x in ["'self'", "'unsafe-inline'", get_static_url_domain(env.get("STATIC_URL", ""))] if x]``
reads a bit better?